basicfileio.c

```diff
@@ -0,0 +1,42 @@
+/*Jinoo Hong*/
+/*Read floats from a .txt file and then output their average as well as squares in another .txt file*/
+#include <stdio.h>
+#include <stdlib.h>
+
+int main () {
```

Comment:
Few things! 
1. This only works for five floats ~
2. This doesn't square the actual float - it only squares the int value and then converts that to a float.